apps/qubit/modules/object/templates/exportResultsSuccess.php

```diff
@@ -0,0 +1,7 @@
+<h1><?php echo __('Export initiated') ?></h1>
+
+<section class="actions">
+  <ul>
+    <li><?php echo link_to(__('Back'), array('module' => 'object', 'action' => 'export'), array('class' => 'c-btn')) ?></li>
+  </ul>
+</section>
```

Comment:
I'd remove this template if you could redirect to the previous page where the export was initiated